packages/babylon/test/fixtures/experimental/uncategorised/62/expected.json

```diff
@@ -0,0 +1,553 @@
+                "start": 1,
+                "end": 9,
+                "loc": {
+                  "start": {
+                    "line": 1,
+                    "column": 1
+                  },
+                  "end": {
+                    "line": 1,
+                    "column": 9
+                  }
+                },
+                "name": "decorate"
+              },
+              "arguments": [
+                {
+                  "type": "ArrowFunctionExpression",
+                  "start": 10,
+                  "end": 23,
+                  "loc": {
+                    "start": {
+                      "line": 1,
+                      "column": 10
+                    },
+                    "end": {
+                      "line": 1,
+                      "column": 23
+                    }
+                  },
+                  "id": null,
+                  "generator": false,
+                  "expression": true,
+                  "params": [
+                    {
+                      "type": "Identifier",
+                      "start": 11,
+                      "end": 14,
+                      "loc": {
+                        "start": {
+                          "line": 1,
+                          "column": 11
+                        },
+                        "end": {
+                          "line": 1,
+                          "column": 14
+                        }
+                      },
+                      "name": "arg"
+                    }
+                  ],
+                  "body": {
+                    "type": "Literal",
+                    "start": 19,
+                    "end": 23,
+                    "loc": {
+                      "start": {
+                        "line": 1,
+                        "column": 19
+                      },
+                      "end": {
+                        "line": 1,
+                        "column": 23
+                      }
+                    },
+                    "value": null,
+                    "rawValue": null,
+                    "raw": "null"
+                  }
+                }
+              ]
+            }
+          }
+        ],
+        "id": {
+          "type": "Identifier",
+          "start": 31,
+          "end": 33,
+          "loc": {
+            "start": {
+              "line": 2,
+              "column": 6
+            },
+            "end": {
+              "line": 2,
+              "column": 8
+            }
+          },
+          "name": "Ex"
+        },
+        "superClass": null,
+        "body": {
+          "type": "ClassBody",
+          "start": 34,
+          "end": 37,
+          "loc": {
+            "start": {
+              "line": 2,
+              "column": 9
+            },
+            "end": {
+              "line": 3,
+              "column": 1
+            }
+          },
+          "body": []
+        }
+      }
+    ]
+  },
+  "comments": [],
+  "tokens": [
```

Comment:
All of these `tokens` should be removed. FWIW, if you run a babylon test fixture and don't have an `expected.json` then it automatically saves one for you with the AST of `actual.js`.